latch_cli/services/register/models.py

```diff
@@ -187,16 +205,19 @@ def image_tagged(self):
         # TODO (kenny): check version is valid for docker
         if self.version is None:
             raise ValueError(
-                "Attempting to create a tagged image name without first"
+                "Attempting to create a tagged image name without first "
                 "extracting the package version."
             )
         if self.image is None or self.version is None:
             raise ValueError(
-                "Attempting to create a tagged image name without first"
+                "Attempting to create a tagged image name without first "
                 " logging in or extracting the package version."
             )
         return f"{self.image}:{self.version}"
 
+    def task_image_name(self, task_name: str) -> str:
+        return f"{self.image_tagged}-{task_name}"
```

Comment:
imo it would make more sense to be `{image_name}-{task_name}:{version}` instead of `{image_name}:{version}-{task_name}`

Reply:
yessir